.github/workflows/build.yml

```diff
@@ -103,10 +104,10 @@ jobs:
           -DFAISS_ENABLE_GPU=${FAISS_ENABLE_GPU} \
           -DFAISS_ENABLE_PYTHON=OFF -DBUILD_TESTING=ON \
           -DFAISS_OPT_LEVEL=${FAISS_OPT_LEVEL} \
-          -DCMAKE_CUDA_ARCHITECTURES="35;52;60;61;70;75" \
+          -DCMAKE_CUDA_ARCHITECTURES="35-real;50-real;60-real;70-real;75" \
           -DCMAKE_BUILD_TYPE=Release
-        make -C build -j4
-        make -C build install
+        cmake --build build --config Release -j2
```

Comment:
what does the release option change? Does it produce faster / smaller binaries?
(I'm doing the same kind of repo for java and I've been taking this repo as a model so I'm quite interested by any such improvement)

Reply:
This doesn't really affect performance, but will drop debugging symbols in the binary, see https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html